### DISCUSSION TITLE
playing multiple videos in frame

### DISCUSSION THREAD

**Discussion by nicolehope5** [jspsych/jsPsych#2662] [Q&A]
I'm trying to create an "odd one out" experiment where 3 videos appear and play one at a time. Is this possible in jsPsych?

**Comment by ChristopheBossens** [comment]
Hi Nicole,

I think the jsPsych video plugin only allows a single video to be played, so you will have to make adaptations to that plugin in order to achieve what you want. Could you clarify if you would like to have three videos that appear simultaneously on the screen (so three video stimuli on the screen), and they play in sequential order. Or do you need a single video that plays three videos after each other?

**Reply by nicolehope5** [reply]
Thank you for your reply! I need the former: I would like to have 3 videos that appear simultaneously on the screen where the video stimuli play in sequential order. However, for my current experiment, I can get away with playing three videos after each other in a single video.

**Comment by robfuscator** [comment]
I didn't test this, but you might be able to use the HTMLKeyboardResponse Plugin to achieve this, and use [HTML video tags](https://www.w3schools.com/tags/tag_video.asp) to include your videos. But this would require some HTML and JavaScript knowledge to make the videos play in sequential order.

Otherwise you could work around it by creating one large video which contains all of the three videos and plays them sequentially.

But these are more ideas than concrete solutions.
